tests/system/marathon_common_tests.py

```diff
@@ -1007,6 +1007,32 @@ def test_metric_endpoint(marathon_service_name):
         "service.mesosphere.marathon.app.count is absent"
 
 
+def test_healtchcheck_and_volume():
+    """Launches a Docker container on Marathon."""
+
+    app_def = apps.healthcheck_and_volume()
+    app_id = app_def["id"]
+
+    client = marathon.create_client()
+    client.add_app(app_def)
+    shakedown.deployment_wait(app_id=app_id)
+
+    tasks = client.get_tasks(app_id)
+    app = client.get_app(app_id)
+
+    assert len(tasks) == 1, "The number of tasks is {} after deployment, but only 1 was expected".format(len(tasks))
+    assert app['container']['type'] == 'DOCKER', "The container type is not DOCKER"
```

Comment:
is this needed? how is supposed to be different from what's in the app definition? :)

Reply:
copied this from somewhere else, will remove it